src/highdicom/seg/sop.py

```diff
@@ -1372,11 +1447,12 @@ def _build_luts(self) -> None:
         index values.
 
         """
+        self._db = sqlite3.connect(":memory:")
```

Comment:
You may need to be careful with the type of the `_db` attribute. We currently don't rely on mypy to check type annotations, but this would probably result in an error. See https://github.com/ImagingDataCommons/dicomweb-client/blob/3f91d8d8059a9f8fe117b1c28581ea814660478e/src/dicomweb_client/file.py#L560-L561

Reply:
Not really sure what the specific concern is here. The linked code appears to need type annotations because the connection variable may be `None` and may change type during the course of the programme to `sqlite3.Connection`. However that is not the case here: the `self._db` is never `None` so the annotation is not needed. I also checked with mypy and it doesn't raise any errors related to this.

If your concern is that the name of the variable is ambiguous, I changed it to `_db_con` in 2d1752e, and also added an explicit type annotation.

Reply:
(I accidentally changed to `db_con` instead of `_db_con` 2d1752e and corrected in b7f232f)